src/2982R0_std_quantity_as_a_numeric_type.md

```diff
@@ -991,18 +993,136 @@ The above program will produce the following types for acceleration quantities
+
+Here are some examples, using Astronomical Units (au), meters (m), degrees (deg), and radians (rad).
+
+| Unit ratio | `std::ratio` representation | vector space representation |
+|-----------|-----------------------------|-----------------------------|
+| $\left(\frac{\text{au}}{\text{m}}\right)$ | `std::ratio<149'597'870'700>` | `magnitude<power_v<2, 2>(), 3, power_v<5, 2>(), 73, 877, 7789>` |
+| $\left(\frac{\text{au}}{\text{m}}\right)^2$ | Unrepresentable (overflow) | `magnitude<power_v<2, 4>(), power_v<3, 2>(), power_v<5, 4>(), power_v<73, 2>(), power_v<877, 2>(), power_v<7789, 2>()>` |
+| $\sqrt{\frac{\text{au}}{\text{m}}}$ | Unrepresentable | `magnitude<2, power_v<3, 1, 2>(), 5, power_v<73, 1, 2>(), power_v<877, 1, 2>(), power_v<7789, 1, 2>()>` |
+| $\left(\frac{\text{rad}}{\text{deg}}\right)$ | Unrepresentable | `magnitude<power_v<2, 2>(), power_v<3, 2>(), power_v<3.14159265358979323851e+0l, -1>(), 5>` |
+
+The variadic magnitude types have one disadvantage: they are more verbose.  We may be able to hide them via
+opaque types with nicer names, using a similar strategy as we have for units.  In any case, their major
+advantage is that they fulfill the requirements stated above --- indeed, they are the only solution we have
+seen which does.
+
+### Construction helpers
+
+We do not want end users to specify the variadic `magnitude` implementation manually.  That would be too labor
+intensive and error prone.  Instead, we provide _construction helpers_ for end users.
+
+`mag<N>` creates the vector space representation of the integer `N`.  This is a simple value, so we can
+multiply and divide it with other magnitudes.
+
+For example, `mag<180> / mag<240>` produces `magnitude<power_v<2, -2>(), 3>`, which is $3/4$.  Notice how the
+result is always automatically represented in lowest terms, because a fraction _not_ in lowest terms cannot be
+represented as a product of powers of primes.
+
+### Common units
 
+Some operations, such as addition or inequality comparison, are "common unit" operations.  In order to execute
+them on quantities of the same dimension, we must first convert them to their _common unit_.
+
+For units which are _commensurable_ --- that is, units whose ratio is a rational number --- the "common unit"
+is the largest unit that evenly divides both.  For example, the common unit of the _foot_ and _inch_ is the
+inch.
+
+This also includes instances where neither unit is an integer multiple of the other.  For example, the common
+unit of the meter and the yard does not have a name, but is equivalent to 800 micrometers.  If we call this
+unit `U`, then there are 1250 `U` per meter, and 1143 `U` per yard.
+
+The practical benefit of this definition is that this unit conversion will simply multiply each participating
+quantity by an exact integer.  If we use integer types to begin with, we can continue to use them without
+losing precision.
```

Comment:
The second paragraph is talking about another concept.
The flow would suggest it should be the same concept.